### DISCUSSION TITLE
Can't view or/and control standalone client' s screen after upgrading to Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by nikalakos** [epoptes/epoptes#176] [Q&A] [answered]
I upgraded a standalone client from Ubuntu 20.04 to 22.04. Server runs also Ubuntu 22.04. After upgrading i "see" that client (in thumbnail view i see the epoptes default red icon(std), not client's active screen), i can shut it down but i can't either monitor or control it (Monitor user, Control user).

**Comment by alkisg** [comment, marked as answer]
Hi nikalakos, epoptes doesn't yet support wayland.
Check if everything works fine in a Xorg session:
https://linuxconfig.org/how-to-enable-disable-wayland-on-ubuntu-22-04-desktop

(which is why we recommend MATE, KDE, XFCE, LXQT etc instead of GNOME)

**Reply by nikalakos** [reply]
Thanks Alkis. I'll check it out.

**Reply by nikalakos** [reply]
Thanks a lot Alkis. It worked!
